app/src/Controller/AppController.php

```diff
@@ -41,13 +41,23 @@ public function initialize(): void
     {
         parent::initialize();
 
-        $this->loadComponent('RequestHandler');
+//         $this->loadComponent('RequestHandler');
```

Comment:
Is there any reason for commented out the RequestHandler component? Please kindly clarify.

Reply:
it was my bad, I was trying to have access to the initial page, I forgot to uncomment. in the end was something else. also now is restored